Assets/Scripts/Gamestate/AIManager.cs

```diff
@@ -18,11 +19,32 @@ public class AIManager : PlayerManager
     [SerializeField]
     private LayerMask ignoreMask;
     public BiddingAI biddingAI;
+    private Rigidbody body;
+    private Collider colliderBox;
+    [SerializeField]
+    private AnimationCurve jumpYoffset;
+    [SerializeField]
+    private float itemStoppingDistance = 0.5f;
+    [SerializeField]
+    private float shootingStoppingDistance = 5f;
+    [SerializeField]
+    private bool updateRotation = true;
```

Comment:
Why is this serialized? This is just a state, right?

Reply:
It's intentional as it's necessary to stop bidding AIs from rotating